helm-charts/nxrm-ha/Chart.yaml

```diff
@@ -34,8 +35,10 @@ keywords:
   - ci
   - repository-manager
   - nexus3
-home: https://www.sonatype.com/nexus-repository-oss
-icon: https://sonatype.github.io/helm3-charts/NexusRepo_Vertical.svg
+home: https://www.sonatype.com/products/sonatype-nexus-repository
+icon: https://www.sonatype.com/hubfs/2019%20Horizontal%20Product%20logo/horizontal%20Repo%20/Repo/NexusRepo_horiz.svg
```

Comment:
I wonder whether we should use links.s.c for these

Reply:
@bobotimi mostly that if these are baked into the builds then it might make sense to create `link.sonatype.com` redirects so we can create permanently working URLs (in theory, but we never verify these and in particular these are pointing at marking URLs which may change).